pkg/framework/plugins/defaultevictor/defaultevictor.go

```diff
@@ -64,6 +65,7 @@ func HaveEvictAnnotation(pod *v1.Pod) bool {
 }
 
 // New builds plugin from its arguments while passing a handle
+// nolint: gocyclo
```

Comment:
@ingvagabund do not want to forget about this. I need to add it to add it to make the linter pass. I saw the codebase has a bunch of this, but if you want to proceed with something else let me know.